.github/CODEOWNERS

```diff
@@ -1,7 +1,4 @@
 # See go/codeowners - automatically generated for confluentinc/schema-registry-images:
-*	@Clarence97
 *	@confluentinc/data-governance
-*	@ConfluentJenkins
-*	@SamehTawfik
-*	@amitkgupta
+*       @Clarence97
 *	@jkao97
```

Comment:
I'd suggest removing all individual people from the file for maintenance purposes.
```suggestion
```

If you do absolutely need them, you need to put them on the same line like
```
*	@confluentinc/data-governance @Clarence97 @jkao97
```
This would mean that each of these users/groups would have permission to approve PRs for all files in this repo. Your current implementation actually only allows `@jkao97` to approve PRs because the last line overrides previous ones.

Reply:
That makes sense, was wondering why only jkao97 was allowed to approve PRs. Made the change to remove all individual people.